x-pack/plugins/fleet/common/errors.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FleetErrorType } from './types';
+
+export class IngestManagerError extends Error {
```

Comment:
I add to move a server side service here, I think it make sense to have errors define here no?